### ISSUE TITLE
Normals and records

### ISSUE THREAD

**Issue by papjul** [open-meteo/open-meteo#361]
Thank you very much for the climate change API, would it be possible to have a more lightweight version with aggregated data by month?

Data I believe would be interesting:

**Normals** for a given month ([30-year period, updated every decade](https://en.wikipedia.org/wiki/Climatological_normal))
What a typical day of June looked like in this 30-year period:
- Average Min temperature
- Average Max temperature
- Average Total of precipitation

**Records** for a given month (all time):
- Min temperature
- Max temperature
- Total of precipitation

Thank you!

**Comment by patrick-zippenfenig** [comment]
Good idea, although I would not offer all-time records like maximum temperature. This information can be very misleading.

Instead the maximum temperature of each month can be calculated and then averaged over 30 years. This should give a good indication of a typical "hot" day in a month

**Comment by papjul** [comment]
Yes, I was not expecting using records as an indication of maximum reachable temperature, but rather knowledge fact, associated with a datetime (example: on 2003-06-XX, temperature reached the all-time record of XX.X °C for a month of June).

But, it's not a must-have to me, I prefer having normals for max temperature and min temperature first ;)

**Comment by wiesehahn** [comment]
Yes, this is also what I would be interested in. I guess there are multiple existing datasets. In ecological modelling e.g. the use of worldclim (and bioclim) for historical (https://www.worldclim.org/data/worldclim21.html) and projected climate data (https://www.worldclim.org/data/cmip6/cmip6climate.html) is quite common.
